[18.0][MIG] partner_email_check: Migration to 18.0

Kimkhoi3010 opened the pull request:
I add 1 PR: https://github.com/OCA/partner-contact/pull/1854
to update res_config_settings_view_form and view_general_configuration

Commit:
Add module partner_contact_birthdate.

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

sysadminmatmoz committed:
Translations and templates sync

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

leemannd committed:
Fix multiple tabs "contact information"

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

leemannd committed:
Port of Birthdate

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

rousseldenis committed:
Add module to manage GLN partner identification number

Commit:
[MIG] Migration partner_contact_birthdate partner_contact_gender part… (#315)

[MIG] Migration partner_contact_birthdate partner_contact_gender partner_contact_personal_information_page to v10

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

pedrobaeza committed:
[FIX] partner_contact: Fix Travis

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

oca-transbot committed:
OCA Transbot updated translations from Transifex

GillesTephaneMeyomesse committed:
[MIG] Partner Identification Gln v10

yajo committed:
[ADD] partner_phonecall_schedule: Know partner's best phonecall time (#475)

yajo committed:
[FIX] partner_phonecall_schedule: Allow partner managers to use addon (#480)

yajo committed:
[MIG] partner_phonecall_schedule: Migrated to v10

Commit:
[MIG] partner_contact_birthdate: Migration to 11.0

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

oca-transbot committed:
OCA Transbot updated translations from Transifex

denislour committed:
Add new module partner-email-check (#496)

* Add new module partner-email-check
Validate email input

oca-transbot committed:
OCA Transbot updated translations from Transifex

Commit:
Migrate module partner_email_check to version 11.0

alexis-via committed:
Add module partner_disable_gravatar

alexis-via committed:
Update to latest OCA coding conventions

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-travis committed:
Update partner_disable_gravatar.pot

oca-travis committed:
[UPD] Update partner_phonecall_schedule.pot

chienandalu committed:
[MIG] partner_phonecall_schedule: Migration to 11.0

rschnapka committed:
Translated using Weblate (German)

Currently translated at 100,0% (9 of 9 strings)

Translation: partner-contact-11.0/partner-contact-11.0-partner_phonecall_schedule
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-11-0/partner-contact-11-0-partner_phonecall_schedule/de/

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

anandkansagra committed:
[MIG] partner_phonecall_schedule: Migration to 12.0.

alexis-via committed:
partner_disable_gravatar from v10 to v12

Olivier-LAURENT committed:
[CHG] remove useless birthdate field; changes after review; fix pylint warning; move birthdate field on the main area of the form

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

OCA-git-bot committed:
README.rst

oca-travis committed:
Update partner_disable_gravatar.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-12.0/partner-contact-12.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_disable_gravatar/

oihane committed:
[MIG] partner_contact_birthdate: Migration to 12.0

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

Commit:
Remove copyright in init + ref code.

OCA-git-bot committed:
icon.png

oca-travis committed:
Update partner_email_check.pot

patrickrwilson committed:
[ADD] partner_priority module

This module adds a priority field to contacts which can be configured within the configuration menu.

[UPD] LINT errors

fixed lint error, renamed test py file, added missing readme

[UPD] Removed unused file

removed incorrect py file

[FIX] wrong sequence val in test

Remove Test

[FIX] Data File Sequence

Fixed wrong sequence field name on data file.

[UPD] noupdate

Removed <data> element and moved noupdate into Odoo as suggested by reviewers.

[UPD] Reviewer Changes

Changes based on reviewer's suggestions

[UPD] Additional Review Changes

Additional changes requested from reviewers

[UPD] Sequence

Moved next sequence out of create method and into field def.

[UPD] Review Change

[FIX] LINT

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update partner_priority.pot

OCA-git-bot committed:
README.rst

OCA-git-bot committed:
[UPD] README.rst

hhgabelgaard committed:
Translated using Weblate (Danish)

Currently translated at 100.0% (2 of 2 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_contact_birthdate/da/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

eduardgm committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_contact_birthdate/es/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

ntsirintanis committed:
Make debugger record a debug message instead of error when importing validate_email in partner_email_check

OCA-git-bot committed:
partner_email_check 11.0.1.0.1

janverb committed:
Add normalization and optional uniqueness to partner_email_check

- partner_email_check now uses email-validator instead of
  validate_email
- Email addresses are normalized
- There is a setting to enforce uniqueness of partner email addresses
- There is a setting to check whether emails are
  deliverable (i.e. whether the domain resolves)

Commit:
[MIG] partner_identification_gln: Migration to 12.0

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_contact_birthdate/pt_BR/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

janverb committed:
Fix partner_email_check conflict with portal, bump version

portal's tests set an invalid email address, `c@c`, that's rejected
by the current version of partner_email_check. If partner_email_check
is installed before portal's tests run then they fail.

To solve this, skip email validation if tests are running, unless a
key in the context is set to make the checks run anyway. That lets
tests in other addons opt in to the checks as well.

The previous update to partner_email_check didn't bump the version
number, so do that now.

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (2 of 2 strings)

Translation: partner-contact-12.0/partner-contact-12.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-partner_contact_birthdate/hr/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

oca-travis committed:
Update partner_email_check.pot

OCA-git-bot committed:
partner_email_check 11.0.2.0.0

newtratip committed:
[13.0][MIG] partner_contact_birthdate

[13.0][UPD] partner_contact_birthdate

[13.0][UPD] Change xml id to view_partner_form

[13.0][UPD] Remove copyright in init files

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] README.rst

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

sergio-teruel committed:
[IMP] partner_phonecall_schedule: black, isort

sergio-teruel committed:
[MIG] partner_phonecall_schedule: Migration to v13.0

newtratip committed:
[IMP] partner_priority: black, isort

newtratip committed:
[13.0][MIG] partner_priority

oca-travis committed:
[UPD] Update partner_priority.pot

ajayjaviya committed:
[IMP] partner_contact_birthday: Add computed age field

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] README.rst

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

skeller1 committed:
[IMP] partner_disable_gravatar: black, isort, prettier

skeller1 committed:
[MIG] partner_disable_gravatar: Migration to 13.0

Commit:
partner_email_check: Migration to 12.0

sbejaoui committed:
[13.0][MIG] - migration partner_identification_gln

sergio-teruel committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_phonecall_schedule
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_phonecall_schedule/es/

oca-travis committed:
[UPD] Update partner_disable_gravatar.pot

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_contact_birthdate/pt_BR/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1 of 1 strings)

Translation: partner-contact-13.0/partner-contact-13.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-13-0/partner-contact-13-0-partner_disable_gravatar/pt_BR/

newtratip committed:
[MIG] partner_contact_birthdate: Migration to 14.0

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

qgroulard committed:
partner_email_check: Migration to 13.0

newtratip committed:
[IMP] partner_priority: black, isort, prettier

newtratip committed:
[MIG] partner_priority: Migration to 14.0

oca-travis committed:
[UPD] Update partner_priority.pot

alexis-via committed:
[MIG] partner_disable_gravatar to v14

alexis-via committed:
partner_disable_gravatar: remove PO and POT files

qgroulard committed:
Remove email from copy data if duplicates not allowed

[REF] Partner email chack params on company

[IMP] email_check

[IMP] test partner_email_check use SavepointCase

qgroulard committed:
Add partner email check syntax flag

oca-travis committed:
Update partner_email_check.pot

OCA-git-bot committed:
README.rst

OCA-git-bot committed:
partner_email_check 13.0.1.1.0

oca-travis committed:
[UPD] Update partner_disable_gravatar.pot

OCA-git-bot committed:
[UPD] README.rst

Zar21 committed:
partner_email_check: black, isort, prettier

Zar21 committed:
partner_email_check: Migration to 14.0

oca-travis committed:
Update partner_email_check.pot

OCA-git-bot committed:
README.rst

OCA-git-bot committed:
partner_email_check 14.0.1.1.1

remi-filament committed:
Added translation using Weblate (French)

remi-filament committed:
Translated using Weblate (French)

Currently translated at 90.4% (19 of 21 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_email_check/fr/

simahawk committed:
partner_identification_gln: migrate to v14

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_contact_birthdate/pt/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

holdenrehg committed:
[IMP] partner_contact_birthdate: 15.0 mig changes

- bump manifest version to 15.0.1.0.0
- remove redundant string attribute from age field, per pylint warning

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] Update partner_contact_birthdate.pot

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

[UPD] README.rst

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

oca-travis committed:
[UPD] Update partner_identification_gln.pot

OCA-git-bot committed:
[UPD] README.rst

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_disable_gravatar/it/

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 42.8% (9 of 21 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_email_check/it/

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (21 of 21 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_email_check/it/

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (21 of 21 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_email_check/it/

Shide committed:
[FIX] [14.0] Removed external dependencies already defined in Odoo base

OCA-git-bot committed:
partner_identification_gln 14.0.1.0.1

OCA-git-bot committed:
update dotfiles [ci skip]

juliap98 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-15.0/partner-contact-15.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-15-0/partner-contact-15-0-partner_contact_birthdate/ca/
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

cesar-tecnativa committed:
[MIG] partner_phonecall_schedule: Migration to 15.0

Commit:
[UPD] Update partner_phonecall_schedule.pot

OCA-git-bot committed:
[UPD] README.rst

dhx committed:
[MIG] partner_priority: Migration to 15.0

Commit:
[UPD] Update partner_priority.pot

OCA-git-bot committed:
[UPD] README.rst

carmenbianca committed:
[MIG] partner_contact_birthdate: Migration to 16.0

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

carmenbianca committed:
[FIX] black, isort, prettier

Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (21 of 21 strings)

Translation: partner-contact-14.0/partner-contact-14.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_email_check/it/

legalsylvain committed:
[MIG] partner_disable_gravatar : port from 14.0 to 16.0
- bump version in manifest
- update translation
- remove useless CONFIGURE.rst file

Commit:
[UPD] Update partner_disable_gravatar.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-16.0/partner-contact-16.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_disable_gravatar/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_contact_birthdate/de/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (1 of 1 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_disable_gravatar/de/

PierrickBrun committed:
Translated using Weblate (French)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_contact_birthdate/fr/

Commit:
[MIG] partner_email_check: migration to 15.0

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_contact_birthdate/hr/

Commit:
[UPD] Update partner_email_check.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
partner_email_check 15.0.1.0.1

diogocsc committed:
[MIG] partner_email_check: Migration to 16.0

victoralmau committed:
[FIX] partner_phonecall_schedule: Change SavepointCase to TransactionCase

OCA-git-bot committed:
partner_phonecall_schedule 15.0.1.0.1

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_contact_birthdate/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_contact_birthdate/es/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Commit:
[MIG] partner_identification_gln: Migration to 16.0

[UPD] Update partner_identification_gln.pot

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (15 of 15 strings)

Translation: partner-contact-15.0/partner-contact-15.0-partner_priority
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-15-0/partner-contact-15-0-partner_priority/es/

OCA-git-bot committed:
[BOT] post-merge updates

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification_gln
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification_gln/

JordiMForgeFlow committed:
[IMP] partner_identification*: move _search_duplicate

OCA-git-bot committed:
[BOT] post-merge updates

josep-tecnativa committed:
[IMP] partner_phonecall_schedule: test performance improvement

OCA-git-bot committed:
[BOT] post-merge updates

Ivorra78 committed:
Added translation using Weblate (Spanish)

Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_identification_gln
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_identification_gln/es/

Commit:
[UPD] Update partner_email_check.pot

OCA-git-bot committed:
[BOT] post-merge updates

remi-filament committed:
Added translation using Weblate (French)

remi-filament committed:
Translated using Weblate (French)

Currently translated at 100.0% (1 of 1 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_disable_gravatar
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_disable_gravatar/fr/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_email_check/es/

Daemo00 committed:
[IMP] partner_contact_birthdate: pre-commit execution

Daemo00 committed:
[MIG] partner_contact_birthdate: Migration to 17.0
Also adopt the module

Daemo00 committed:
[REF] partner_contact_birthdate: Remove deprecation warning

NICO-SOLUTIONS committed:
[IMP] partner_identification_gln: pre-commit auto fixes

NICO-SOLUTIONS committed:
[MIG] partner_identification_gln: Migration to 17.0

pedroguirao committed:
[MIG] partner_phonecall_schedule: pre-commit stuff

[MIG] partner_phonecall_schedule: Migration to 16.0

pedroguirao committed:
[MIG] partner_phonecall_schedule: Migration to 16.0

Commit:
[UPD] Update partner_phonecall_schedule.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (9 of 9 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_phonecall_schedule
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_phonecall_schedule/it/

youelEl committed:
[IMP] partner_email_check: pre-commit auto fixes

youelEl committed:
[MIG] partner_email_check: Migration to 17.0

Commit:
[UPD] Update partner_email_check.pot

OCA-git-bot committed:
[BOT] post-merge updates

remi-filament committed:
[IMP] partner_disable_gravatar: apply pre-commit auto fixes

remi-filament committed:
[MIG] partner_disable_gravatar: Migration to 17.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-17.0/partner-contact-17.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-17-0/partner-contact-17-0-partner_email_check/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: partner-contact-17.0/partner-contact-17.0-partner_email_check
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-17-0/partner-contact-17-0-partner_email_check/it/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (9 of 9 strings)

Translation: partner-contact-16.0/partner-contact-16.0-partner_phonecall_schedule
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-16-0/partner-contact-16-0-partner_phonecall_schedule/pt_BR/

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-17.0/partner-contact-17.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-17-0/partner-contact-17-0-partner_contact_birthdate/sv/

Commit:
[UPD] Update partner_identification_gln.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-17.0/partner-contact-17.0-partner_identification_gln
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-17-0/partner-contact-17-0-partner_identification_gln/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: partner-contact-17.0/partner-contact-17.0-partner_identification_gln
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-17-0/partner-contact-17-0-partner_identification_gln/it/

Commit:
[IMP] partner_contact_role: pre-commit execution

Commit:
[IMP] partner_contact_role: pre-commit execution

Kimkhoi3010 committed:
[IMP] partner_email_check: pre-commit auto fixes

Kimkhoi3010 committed:
[MIG] partner_email_check: Migration to 18.0

chaule97 commented:
Please add my PR https://github.com/OCA/partner-contact/pull/1854

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+
+import logging
+
+from odoo import _, api, models
```
```suggestion
from odoo import api, models
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+        validate_email,
+    )
+except ImportError:
+    _logger.debug(_("Cannot import 'email_validator'."))
```
```suggestion
    _logger.debug(self.env._("Cannot import 'email_validator'."))
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+            for rec in self.filtered("email"):
+                if "," in rec.email:
+                    raise UserError(
+                        _(
```
```suggestion
                        self.env._(
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+                    )
+                if self.search_count([("email", "=", rec.email), ("id", "!=", rec.id)]):
+                    raise UserError(
+                        _("Email '%s' is already in use.") % rec.email.strip()
```
```suggestion
                        self.env._("Email '%s' is already in use.", rec.email.strip())
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+            )
+        except EmailSyntaxError:
+            raise ValidationError(
+                _("%s is an invalid email") % email.strip()
```
```suggestion
                self.env._("%s is an invalid email", email.strip())
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -0,0 +1,102 @@
+            ) from EmailSyntaxError
+        except EmailUndeliverableError:
+            raise ValidationError(
+                _("Cannot deliver to email address %s") % email.strip()
```
```suggestion
                self.env._("Cannot deliver to email address %s", email.strip())
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -44,15 +44,15 @@ def _check_email_unique(self):
                 if self.search_count([("email", "=", rec.email), ("id", "!=", rec.id)]):
                     raise UserError(
-                        _("Email '%s' is already in use.") % rec.email.strip()
+                        self.env._("Email '%s' is already in use.") % rec.email.strip()
```
```suggestion
                        self.env._("Email '%s' is already in use.", email.strip())
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -72,11 +72,11 @@ def _normalize_email(self, email):
         except EmailSyntaxError:
             raise ValidationError(
-                _("%s is an invalid email") % email.strip()
+                self.env._("%s is an invalid email") % email.strip()
```
```suggestion
                self.env._("%s is an invalid email", email.strip())
```

chaule97 commented on partner_email_check/models/res_partner.py:
```diff
@@ -72,11 +72,11 @@ def _normalize_email(self, email):
         except EmailUndeliverableError:
             raise ValidationError(
-                _("Cannot deliver to email address %s") % email.strip()
+                self.env._("Cannot deliver to email address %s") % email.strip()
```
```suggestion
                self.env._("Cannot deliver to email address %s", email.strip())
```

chaule97 reviewed (approved):
Pre-approve:

Please add my PR in your Description

BT-mchechlacz committed:
[MIG] partner_multi_relation: Migration to 18.0

Borruso committed:
[MIG] partner_contact_birthdate: Migration to 18.0

lef-adhoc committed:
[IMP] partner_vat_unique: add configuration parameter for unique VAT constraint

- Defined `partner_vat_unique` as a boolean configuration parameter.
- Updated `res.partner` to validate VAT uniqueness based on this parameter.
- Added a new section in ResConfigSettings for enabling/disabling the constraint.
- Updated tests to include scenarios with the configuration parameter.

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_title_order
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_title_order/it/

JordiMForgeFlow committed:
[MIG] partner_identification_gln: Migration to V18

OCA-git-bot committed:
Merge PR #1899 into 18.0

Signed-off-by mmequignon

Commit:
[UPD] Update partner_contact_access_link.pot

JordiMForgeFlow committed:
[IMP] partner_identification_gln: consider different types of errors

The stdnum library may trigger 3 different types of exceptions
(see https://github.com/arthurdejong/python-stdnum/blob/02186014334be2cd2d8899835b8380ba1af74b7b/stdnum/ean.py#L50)

OCA-git-bot committed:
[BOT] post-merge updates

JordiMForgeFlow committed:
[FIX] partner_identification: remove extra character wrongly printed in error message

OCA-git-bot committed:
Merge PR #1870 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update partner_company_group.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1927 into 18.0

Signed-off-by sebalix

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_contact_access_link
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_contact_access_link/

OCA-git-bot committed:
Merge PR #1935 into 18.0

Signed-off-by JordiBForgeFlow

Commit:
[UPD] Update partner_identification.pot

Commit:
[UPD] Update partner_identification_gln.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_identification/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (80 of 80 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_identification/it/

OCA-git-bot committed:
Merge PR #1900 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_contact_personal_information_page.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_contact_personal_information_page
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_contact_personal_information_page/

OCA-git-bot committed:
Merge PR #1923 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_fax.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_fax
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_fax/

Commit:
[UPD] Update partner_disable_gravatar.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1877 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update partner_stage.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_stage
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_stage/

OCA-git-bot committed:
Merge PR #1896 into 18.0

Signed-off-by HaraldPanten

Commit:
[UPD] Update partner_industry_secondary.pot

OCA-git-bot committed:
[BOT] post-merge updates

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 57.5% (46 of 80 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_identification/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 98.7% (79 of 80 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_identification
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_identification/pt/

juancarlosonate-tecnativa committed:
[IMP] partner_priority: pre-commit auto fixes

juancarlosonate-tecnativa committed:
[MIG] partner_priority: Migration to 18.0

OCA-git-bot committed:
Merge PR #1933 into 18.0

Signed-off-by NL66278

Commit:
[UPD] Update partner_multi_relation.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_multi_relation
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_multi_relation/

OCA-git-bot committed:
Merge PR #1964 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_priority.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_priority
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_priority/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_priority
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_priority/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_industry_secondary
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_industry_secondary/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (96 of 96 strings)

Translation: partner-contact-18.0/partner-contact-18.0-partner_multi_relation
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_multi_relation/it/

OCA-git-bot committed:
Merge PR #1902 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_contact_birthdate.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: partner-contact-18.0/partner-contact-18.0-partner_contact_birthdate
Translate-URL: https://translation.odoo-community.org/projects/partner-contact-18-0/partner-contact-18-0-partner_contact_birthdate/

alexis-via committed:
[IMP] partner_disable_gravatar: pre-commit auto fixes

alexis-via committed:
[MIG] partner_disable_gravatar to v18

OCA-git-bot committed:
[IMP] update dotfiles

OCA-git-bot committed:
Merge PR #1940 into 18.0

Signed-off-by alexis-via

Commit:
[UPD] Update partner_disable_gravatar.pot

OCA-git-bot committed:
[BOT] post-merge updates

BhaveshHeliconia committed:
[IMP] partner_phonecall_schedule: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] partner_phonecall_schedule: Migration to 18.0

OCA-git-bot committed:
Merge PR #1906 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_vat_unique.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2004 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_phonecall_schedule.pot

OCA-git-bot committed:
[BOT] post-merge updates
